packages/web3-eth-personal/package.json

```diff
@@ -22,12 +22,12 @@
         "web3-core-method": "1.0.0-beta.52",
         "web3-net": "1.0.0-beta.52",
         "web3-providers": "1.0.0-beta.52",
-        "web3-utils": "1.0.0-beta.52"
+        "web3-utils": "1.0.0-beta.52",
+        "web3-eth-accounts": "1.0.0-beta.52"
```

Comment:
``eth-accounts`` should be a devDependency.

Reply:
https://github.com/ethereum/web3.js/pull/2660#discussion_r275373009

Reply:
Thinking about this more we may as well move `Accounts` interface itself `web3-core` so it can be shared without adding any dependencies or not supplying solid typing's... thoughts?

Reply:
yes right.

Reply:
Yes, we should probably move it to the web3-core module. This issue will solve this problem later in a more clean way: https://github.com/ethereum/web3.js/issues/2109

Reply:
Yes right. @nivida.

Reply:
@nivida I think we should keep web3-eth-accounts as a dependency for now as if I move Accounts in core-module for example.
https://github.com/ethereum/web3.js/blob/1.0/packages/web3-eth-accounts/README.md#usage

We should think for a good solution to solve this.